### ISSUE TITLE
Link PDFs of slideshows on UDLS page

### ISSUE THREAD

**Issue by jakobottar** [ksoc-gradsac/ksoc-gradsac.github.io#19]
need to move storage off-site to B2 or S3 buckets, storage cost will grow too great for GitHub

**Comment by rsmath** [comment]
we'd have to pay eventually to store pdfs here?

**Comment by jakobottar** [comment]
no, but github is not the place to store lots of large files. IDK what the total repo cap is, but the max filesize is 100 MB

**Comment by rsmath** [comment]
Ah, I see.
